src/components/Letter.tsx

```diff
@@ -81,23 +72,18 @@ export function Letter({ letter, shared, isEditable, disableDrag }: Props) {
       defaultPosition={position}
       onStart={() => {
         setDragging(true);
-        // TODO: update z-index to be the highest.
+        setZIndex(100);
       }}
       onStop={(_, dragData) => {
-        // TODO: persist z-index
         localStorage.setItem(
           id,
           JSON.stringify({
             x: dragData.x,
             y: dragData.y,
           })
         );
-        shared?.set(id, {
-          ...currentSharedData,
-          numDrags: currentSharedData.numDrags + 1,
-        });
-
         setDragging(false);
+        setZIndex(0);
```

Comment:
oh i dont think we should reset Z index after, cuz then it would fall to the bottom and you'd just lose it